packages/duplex-channel/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@effection/duplex-channel",
+  "version": "2.0.0-beta.1",
+  "description": "A bidirectional channel for effection",
+  "main": "dist-cjs/index.js",
+  "module": "dist-esm/index.js",
+  "types": "dist-esm/index.d.ts",
+  "typeDocEntry": "src/index.ts",
+  "sideEffects": false,
+  "homepage": "https://github.com/thefrontside/effection",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/thefrontside/effection.git",
+    "directory": "packages/channel"
+  },
+  "author": "Frontside Engineering <engineering@frontside.io>",
+  "license": "MIT",
+  "files": [
+    "README.md",
+    "CHANGELOG.md",
+    "dist-*/**/*",
+    "src/**/*"
+  ],
+  "scripts": {
+    "lint": "eslint '{src,tests}/**/*.ts'",
+    "test": "mocha -r ts-node/register test/**/*.test.ts",
+    "prepack": "tsc --build tsconfig.esm.json && tsc --build tsconfig.esm.json && tsc --build tsconfig.cjs.json",
+    "mocha": "mocha -r ts-node/register"
+  },
+  "devDependencies": {
+    "@effection/mocha": "^2.0.0-beta.9",
+    "@frontside/tsconfig": "^1.2.0",
+    "@types/mocha": "^7.0.1",
+    "@types/node": "^13.13.4",
+    "expect": "^25.4.0",
+    "mocha": "^8.3.1",
+    "ts-node": "^10.1.0",
+    "typescript": "^4.3.5"
+  },
+  "dependencies": {
+    "@effection/core": "^2.0.0-beta.10",
+    "@effection/channel": "^2.0.0-beta.12",
+    "@effection/subscription": "^2.0.0-beta.12"
+  },
```

Comment:
I'm beginning to feel like these "addon" packages like "atom", "fetch", "websocket-client", etc.. should have `effection` as a peer dependency, but that "core" packages have a pinned versions to each other in normal dependencies. With `npm` and `yarn2` now installing a matching peer dependency if it is not present, this should make managing your lock file simpler.

At the very least, I think it is safe to have it depend on a single `effection` version, since it isn't core.

Reply:
Hmm, that's a good point. Is this something we want to tackle for this PR, or something we want to adjust for all of these packages later on?

Reply:
I’m fine with making it follow-on.